examples/gno.land/r/demo/disperse/util.gno

```diff
@@ -0,0 +1,62 @@
+package disperse
+
+import (
+	"std"
+	"strconv"
+	"strings"
+	"unicode"
+)
+
+func parseAddresses(addresses string) ([]std.Address, error) {
+	var ret []std.Address
+
+	for _, str := range strings.Split(addresses, ",") {
+		addr := std.Address(str)
+		if !addr.IsValid() {
+			return nil, ErrInvalidAddress
+		}
```

Comment:
In Go/Gno, its good practice to add a new line after a block, like this: 
```suggestion
		}
		
```

Could you apply this style across the board in the app?

Reply:
https://github.com/gnolang/gno/pull/2613/commits/0a87b66f6037f03a688651a62589e878f1af161b